js/packages/seed-vault/android/gradle.properties

```diff
@@ -1,5 +1,5 @@
-SolanaMobileSeedVaultLibModule_kotlinVersion=1.7.0
-SolanaMobileSeedVaultLibModule_minSdkVersion=21
-SolanaMobileSeedVaultLibModule_targetSdkVersion=33
-SolanaMobileSeedVaultLibModule_compileSdkVersion=33
-SolanaMobileSeedVaultLibModule_ndkversion=21.4.7075529
+SolanaMobileSeedVaultLibModule_kotlinVersion=1.9.0

```

Comment:
weird diff. only change here is kotlin version to 1.9.0.

Note: android targetSDK is held at 33 as this is coupled with react native version and I dont want to raise the required RN version